-Failed tests
Test 1: Teste 1
Test 2: Teste 2
-Test 1: Teste 1 (-10.000)
Incorrect program result
 --- Input ---
> 3
>10
>2
>3
>2
>7
>2
>4
>5
>11

 --- Program output ---
>Tamanho da matriz:Valor:Valor:Valor:Valor:Valor:Valor:Valor:Valor:Valor:Traceback (most recent call last):
>  File "diagonaldominante.py", line 28, in <module>
>    b=soma(a)
>  File "diagonaldominante.py", line 8, in soma
>    for j in range(0,a.shape[1],0):
>ValueError: range() arg 3 must not be zero

 --- Expected output (text)---
> 'SIM'

-Test 2: Teste 2 (-10.000)
Incorrect program result
 --- Input ---
> 3
>5
>2
>3
>2
>3
>2
>4
>5
>8

 --- Program output ---
>Tamanho da matriz:Valor:Valor:Valor:Valor:Valor:Valor:Valor:Valor:Valor:Traceback (most recent call last):
>  File "diagonaldominante.py", line 28, in <module>
>    b=soma(a)
>  File "diagonaldominante.py", line 8, in soma
>    for j in range(0,a.shape[1],0):
>ValueError: range() arg 3 must not be zero

 --- Expected output (text)---
> 'NAO'

-Summary of tests
>+------------------------------+
>|  2 tests run/ 0 tests passed |
>+------------------------------+

